spec/requests/portfolios_spec.rb

```diff
@@ -52,6 +52,24 @@
       end
     end
 
+    describe "POST admin tagged /portfolios/:portfolio_id/portfolio_items" do
+      let(:portfolio_without_item) { create(:portfolio) }
```

Comment:
@lindgrenj6 - I think creating vars `:portfolio` and on line 57 `:portfolio_item` may lead to less confusion for me.